main/SAPI.c

```diff
@@ -69,6 +69,7 @@ SAPI_API void sapi_startup(sapi_module_struct *sf)
 {
 	sf->ini_entries = NULL;
 	sapi_module = *sf;
+	CG(isolate_symbols) = (sf->flags & SAPI_MODULE_FLAG_ISOLATE_SYMBOLS) != 0;
```

Comment:
I meant assignment to ``CG(isolate_symbols)`` in ``php_apache_server_startup()``.
This way we don't need ``STANDARD_SAPI_MODULE_PROPERTIES_WITH_FLAGS`` and other modifications in SAPI layer.

Reply:
@dstogov Temporarily reverted back to using a new SAPI module property, as it turns out changing the value of compiler globals inside is impossible from SAPI activation/startup functions, as when ZTS is in use they are always called before initialization of the thread-safe storage for said compiler globals.